### DISCUSSION TITLE
Anyway to add Netflix order in show format?

### DISCUSSION THREAD

**Discussion by papampi** [pymedusa/Medusa#9819] [Ideas & Feature requests]
Some shows like "Money Heist" have Netflix order in thetvdb instead of dvd order and not show up correctly in Medusa, Is there anyway to make them show up correctly with netflix format?
If not, would be a good idea to add Netflix format in show format next to dvd order.

**Comment by p0psicles** [comment]
We're not going to put any more time in tvdb. But maybe if tvmaze or tmdb supports it, sure

**Comment by papampi** [comment]
TMDB has a "Parts" section which is exactly how Netflix streamed it.
https://www.themoviedb.org/tv/71446-la-casa-de-papel/episode_group/5eb730dfca7ec6001f7beb51

**Reply by p0psicles** [reply]
Ow that's cool. We can implement that in the future. To bad tmdb doesn't do air time.

**Comment by papampi** [comment]
TVMAZE Also has alternate language section which will fix Netflix order 
https://www.tvmaze.com/alternatelists/17/la-casa-de-papel-english-language-premiere
